.github/workflows/cd.yaml

```diff
@@ -113,3 +115,40 @@ jobs:
               with:
                   tag_name: ${{ env.PACKAGE_VERSION_TAG }}
                   release_name: ${{ env.PACKAGE_VERSION }}
+
+            - name: Check out main repo
+              if: steps.version.outputs.IS_NEW_VERSION == 'true'
+              uses: actions/checkout@v2
+              with:
+                  repository: 'PostHog/posthog'
+                  path: 'posthog/'
+
+            - name: Install new plugin server version in main repo
+              if: steps.version.outputs.IS_NEW_VERSION == 'true'
+              run: |
+                  cd posthog/plugins/
+                  jq '.dependencies."@posthog/plugin-server" = "${{ steps.version.outputs.PACKAGE_VERSION }}"' package.json | sponge package.json
+                  yarn
+
+            - name: Create main repo pull request
+              if: steps.version.outputs.IS_NEW_VERSION == 'true'
+              id: main-repo-pr
+              uses: peter-evans/create-pull-request@v3
+              with:
+                  path: posthog/
+                  commit-message: Update plugin server to ${{ steps.version.outputs.PACKAGE_VERSION }}
+                  committer: PostHog Bot <hey@posthog.com>
+                  branch: plugin-server-${{ steps.version.outputs.PACKAGE_VERSION_TAG }}
+                  delete-branch: true
+                  title: Update plugin server to ${{ steps.version.outputs.PACKAGE_VERSION }}
+                  body: |
+                      ## Changes
+
+                      Plugin server version ${{ steps.version.outputs.PACKAGE_VERSION }} has been released. This updates PostHog to use it.
+
+                      [npm releases](https://www.npmjs.com/package/@posthog/plugin-server?activeTab=version) • [GitHub releases](https://github.com/PostHog/plugin-server/releases) • [commit history](https://github.com/PostHog/plugin-server/commits/)
```

Comment:
Would it be hard to add a link like this to the body?

https://github.com/PostHog/plugin-server/compare/v0.8.2...v0.8.3

Reply:
Actually not